docs/content/guides/rate-limiting.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Rate Limiting
-sidebar_position: 3
+title: Per-user Rate Limiting
```

Comment:
Not changing the file name since it is being referred at many places, across websites/products/blogs - ad-hoc docs.

Reply:
Raised a pr for redirect, making changes to the file name as well.